### PR TITLE
Add company_id to matches, fix bugs

### PR DESCRIPTION
**Pull request by ragulanr**
Permit arrays in params[:match]
 Set company_id when creating match using
 current user
 Fix event description
 Render match_id to avoid 500

@anandhs29 
@NirujanSuresh 
@akif-hossain

### CODE REVIEW

**Comment by anandhs29** [comment]
Can you also add another flag on the match, which signifies if the match is a sum of all rounds, or is a best of all rounds

**Comment by anandhs29** [comment]
I'll also need two more array type entries on the match to signify the sides. Similar to winners and losers. Currently I cannot reconstruct the sides from the original request.

File: app/controllers/matches_controller.rb [hunk 1]
```diff
@@ -20,17 +20,20 @@ def show
 	def create_match
 		rounds = params[:rounds]
 		params[:match][:date] = Time.now
+		params[:match][:company_id] = @current_user.company_id
+		puts match_params
```
Comment (akif-hossain) [inline app/controllers/matches_controller.rb]: remove debugging statements?

File: app/controllers/matches_controller.rb [hunk 2]
```diff
@@ -59,8 +64,7 @@ def matches_hash(matches)
 	end
 
 	def match_params
-		params[:match].permit(:players, :winners, :losers, :winning_score, :losing_score,
-			:game_type, :date, :company_id)
+		params[:match].permit(:winning_score, :losing_score, :game_type, :date, :company_id, :players => [], :winners => [], :losers => [])
```
Comment (akif-hossain) [inline app/controllers/matches_controller.rb]: you need to accept round param here. otherwise you wont see it.

**Pull request merged**